cogs/help.py

```diff
@@ -0,0 +1,167 @@
+        self.filter_commands = filter_commands
+        self.prefix = prefix
+        self.menu_author = author
+        super().__init__([(cog, cogs[cog]) for cog in sorted(cogs, key=lambda cog: cog.qualified_name if cog else "ZZ")], per_page=1)
+
+    async def format_page(self, menu: menus.MenuPages, cog_tuple: tuple) -> discord.Embed:
+        cog, commands = cog_tuple
+        embed = discord.Embed(
+            title=f"Help for {cog.qualified_name if cog else 'unclassified commands'}",
+            description=(
+                'Command syntax : `<Those arguments are required>`. `[Those aren\'t]`\n'
+                f'[Everything to know about my glorious self]({constants.invite_link} "Invite link")\n'
+                f'The prefix for this channel is `{self.prefix}`\n{cog.description if cog else ""}'
+            ),
+            colour=0xffff00,
+        )
+        embed.set_author(
+            name=self.menu_author.display_name,
+            icon_url=str(self.menu_author.avatar_url),
+        )
+        for command in await self.filter_commands(commands):
+            embed.add_field(
+                name=f"{self.prefix}{self.get_command_signature(command)}",
+                value=command.help,
+                inline=False,
+            )
+        embed.set_footer(text=f"Page {menu.current_page + 1}/{self.get_max_pages()}")
+        return embed
+
+
+class Help(commands.HelpCommand):
+    def get_command_signature(self, command: commands.Command) -> str:
+        basis = f"{command.qualified_name}"
+        for arg in command.clean_params.values():
+            if arg.kind in (Parameter.VAR_KEYWORD, Parameter.VAR_POSITIONAL):
+                basis += f" [{arg.name}]"
+            elif arg.annotation == Optional:
+                basis += f" [{arg.name} = None]"
+            elif isinstance(arg.annotation, commands.converter._Greedy):
+                basis += f" [{arg.name} = (...)]"
+            elif arg.default == Parameter.empty:
+                basis += f" <{arg.name}>"
+            else:
+                basis += f" [{arg.name} = {arg.default}]"
+        return basis
+
+    async def send_bot_help(self, mapping: dict) -> None:
+        ctx = self.context
+        pages = menus.MenuPages(
+            source=HelpSource(
+                self.get_command_signature,
+                self.filter_commands,
+                ctx.prefix,
+                ctx.author,
+                mapping,
+            ),
+            clear_reactions_after=True,
+        )
+        await pages.start(ctx)
+
+    async def send_cog_help(self, cog: commands.Cog) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=cog.qualified_name,
+            description=(
+                f"Command syntax : `<Those arguments are required>`. `[Those aren't]`\n"
+                f"The prefix for this channel is `{prefix}`\n{cog.description}"
+            ),
```

Comment:
Same as the above suggestion:
```suggestion
            description=textwrap.dedent(
                f"""
                Help syntax : `<Required argument>`. `[Optional argument]`
                Command prefix: `{self.prefix}`
                {cog.description}
                """
            ),
```